_posts/2017-03-07-hddtools.md

```diff
@@ -0,0 +1,323 @@
+---
```

Comment:
Thanks for this @cvitolo! A really nice addition to the blog series on onboarded software. I've added my comments but before making any changes, please wait for @sckott to chime in.